app/views/orgs/lti_configurations/info.html.erb

```diff
@@ -9,12 +9,12 @@
     </div>
     <div class="site-content-body markdown-body">
       <p>Connecting GitHub Classroom to <%= lms_name %> will allow you to manage your students and submissions easier!</p>
-      <h3>What integrating Classroom with <%= lms_name %> can do for you</h3>
+      <h3>What integrating GitHub Classroom with <%= lms_name %> can do for you</h3>
       <ul>
-        <li>Sync rosters from <%= lms_name %> to Classroom</li>
+        <li>Sync rosters from <%= lms_name %> to GitHub Classroom</li>
```

Comment:
Before this patch, this would have said "Sync rosters from Google Classroom to Classroom" 😱